freeprocess.mod/freeprocess.c

```diff
@@ -268,6 +279,18 @@ int fdTerminateProcess( int pid ){
 	return res;
 }
 
+//returns 0 for success
+int fdKillProcess( int pid ){
+	PROCESS_INFORMATION *pi=(PROCESS_INFORMATION *)pid;

```

Comment:
On initial scan, that doesn't look right - extracting a struct pointer from an Int. Me thinks this may be a problem on 64-bit.
If there's other code already like that, then of course, there's probably a but elsewhere already...

Reply:
Yes I did not change anything except function names (I cut from fdTerminateProcess - and pasted to fdKillProcess).
I already wondered when reading "int" but did not want to do experiments there ;-)